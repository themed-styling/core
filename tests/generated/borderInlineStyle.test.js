import 'jest-styled-components'

import { borderInlineStyle } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderInlineStyle, 'borderInlineStyle')
