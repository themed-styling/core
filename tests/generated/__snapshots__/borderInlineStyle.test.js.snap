// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderInlineStyle renders 0 1`] = `
Array [
  .c0 {
  border-inline-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders function 1`] = `
Array [
  .c0 {
  border-inline-style: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders invalid theme path 1`] = `
Array [
  .c0 {
  border-inline-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders negative number 1`] = `
Array [
  .c0 {
  border-inline-style: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-inline-style: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-inline-style: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-inline-style: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-inline-style: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-inline-style: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-inline-style: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-inline-style: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    border-inline-style: July;
  }
}

@media April {
  .c0 {
    border-inline-style: August;
  }
}

@media January {
  .c0 {
    border-inline-style: September;
  }
}

@media April {
  .c0 {
    border-inline-style: November;
  }
}

@media June {
  .c0 {
    border-inline-style: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-inline-style: July;
  }
}

@media April {
  .c0 {
    border-inline-style: August;
  }
}

@media January {
  .c0 {
    border-inline-style: September;
  }
}

@media April {
  .c0 {
    border-inline-style: November;
  }
}

@media June {
  .c0 {
    border-inline-style: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-inline-style: January;
  }
}

@media April {
  .c0 {
    border-inline-style: December;
  }
}

@media January {
  .c0 {
    border-inline-style: April;
  }
}

@media April {
  .c0 {
    border-inline-style: May;
  }
}

@media June {
  .c0 {
    border-inline-style: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-inline-style: January;
  }
}

@media April {
  .c0 {
    border-inline-style: December;
  }
}

@media January {
  .c0 {
    border-inline-style: April;
  }
}

@media April {
  .c0 {
    border-inline-style: May;
  }
}

@media June {
  .c0 {
    border-inline-style: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders postitive number 1`] = `
Array [
  .c0 {
  border-inline-style: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders string 1`] = `
Array [
  .c0 {
  border-inline-style: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderInlineStyle renders valid theme path 1`] = `
Array [
  .c0 {
  border-inline-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-inline-style: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
